CMakeLists.txt

```diff
@@ -0,0 +1,35 @@
+cmake_minimum_required(VERSION 3.16)
+project(evm_bridge_contracts)
+add_subdirectory (solidity)
+include(ExternalProject)
+# if no cdt root is given use default path
+if(CDT_ROOT STREQUAL "" OR NOT CDT_ROOT)
```

Comment:
I don't think this is necessary. We don't need this for system contracts and can still direct `find_package(cdt)` to a local build just fine.